### ISSUE TITLE
Selecting a theme and CMS for the website

### ISSUE THREAD

**Issue by pzwsk** [GFDRR/rdl-website-dev#1]
Hi, I had a look to CMS solutions for Jekyll-based Github-hosted websites. You have several good solutions including Netlify CMS, Forestry, Siteleaf or Cannon CMS. See the list here https://jekyllthemes.io/resources/jekyll-hosting-and-cms-solutions

I would incline to keep using Forestry.io. Here is why:

- When looking for a way to rebuild the website into a fully responsive version, I found uBuild jekyll theme https://forestryio.github.io/ubuild-jekyll/. 

- uBuild works with the concept of layout as blocks. Each block is defined by an html file. When you configure the layout of a page, you just add the blocks in the order you want. And if you want to add or edit a block, you edit its html file and related css.

- Then, you can add or edit the content of your pages through Forestry.io CMS or directly in the files. Each file is written in YAML and most of other Jekyll CMS support YAML. 

- So in summary, uBuild theme as the advantage of being fully responsive, very easy to modify and can be used through Forestry.io CMS right away or through other jekyll CMS subject to some configuration with YAML files.

- The last argument is that we already have the layout converted to uBuild here http://dev.riskdatalibrary.org/ and modifying it would be quite easy for any web dev with basic experience in Jekyll.

So if ok, I will proceed with the

**Comment by ConnectedSystems** [comment]
Hi @pzwsk 

My only concern with forestry is that they are free only for personal use. Given this a GFDRR project, it would incur a $29/month charge, at least (https://forestry.io/pricing/). 

Do note that this plan level does not allow for subdomains (the next plan level does, but costs $749/month).
This is only an issue if we plan to move all sites across to Forestry (away from being hosted by GitHub Pages)

If this is acceptable, then I too would prefer we continue with the initial implementation.

**Comment by pzwsk** [comment]
Hi @ConnectedSystems 

My understanding is that we can use the free plan even for GFDRR projects, see below:

![image](https://user-images.githubusercontent.com/2378439/110434354-fa30c900-80b1-11eb-8e28-298b09ca540d.png)

**Comment by ConnectedSystems** [comment]
In that case, please disregard my comments above.

**Comment by pzwsk** [comment]
This was approved during weekly call so closing.
